test/src/comps/test_determiter.cc

```diff
@@ -1,7 +1,7 @@
+#include <gtest/gtest.h>
```

Comment:
Previously I got build errors when I included gtest before certain other libraries.  Are you moving it here for style reasons?

Reply:
nope, to resolve build errors when it is included after lapackpp.